.circleci/config.yml

```
@@ -37,7 +37,7 @@ jobs:
     steps:
       - checkout
       - restore-cache: *restore-cache
-      - run: yarn --no-progress --ignore-engines
+      - run: yarn --no-progress --ignore-engines --frozen-lockfile
```

```suggestion
      - run: yarn --no-progress --frozen-lockfile --ignore-engines
```
to keep it visible what's special about this one :)